docs/resources/conversation.md

```diff
@@ -68,6 +87,15 @@ conversation should be archived or left behind on destroy. Valid values are
 `archive | none`. Note that when set to `none` the conversation will be left
 as it is  and as a result any subsequent runs of terraform apply with the same
 name  will fail.
```

Comment:
Note that I think, regardless of whether archive or none is specified, subsequent applies will fail unless adopt_existing_channel is set to true.